ryhope/src/storage/pgsql/storages.rs

```diff
@@ -770,13 +802,79 @@ impl<T: Debug + Clone + Send + Sync + Serialize + for<'a> Deserialize<'a>> Cache
         self.dirty = false;
         self.in_tx = false;
     }
+
+    async fn fetch_at_inner(&self, epoch: IncrementalEpoch) -> T {
+        trace!("[{self}] fetching payload at {}", epoch);
+        let connection = self.db.get().await.unwrap();
```

Comment:
Can you put `expect()` instead.

Reply:
Done in commit https://github.com/Lagrange-Labs/mapreduce-plonky2/pull/425/commits/23affb5f3b0c62fc1ab825043196a2f89e20039c